features/layout/sidebar-navigation/sidebar-navigation.tsx

```diff
@@ -164,7 +197,9 @@ export function SidebarNavigation() {
           <Logo
             src={
               isSidebarCollapsed
-                ? "/icons/logo-small.svg"
+                ? isMobileScreen
+                  ? "/icons/logo-large.svg"
+                  : "/icons/logo-small.svg"
```

Comment:
Here are two intersting solutions that don't require JS "media queries"
https://github.com/profydev/prolog-app-MarkusB2611/pull/7/files
https://github.com/profydev/prolog-app-daniel-moderiano/pull/6/files